packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -195,7 +195,17 @@ export class GeneratedAnswerCommon {
     const {citations} = getGeneratedAnswerState() ?? {};
     const {logCitationHover} = getGeneratedAnswer() ?? {};
 
-    return citations?.map(
+    const uniqueCitations = citations?.reduce(
+      (acc: GeneratedAnswerCitation[], citation) => {
+        if (!acc.some((item) => item?.clickUri === citation.clickUri)) {
+          acc.push(citation);
+        }
+        return acc;
+      },
+      []
+    );
+
+    return uniqueCitations?.map(
```

Comment:
Double array iterations.
An Array.flatMap should do the job with a single array iteration